### ISSUE TITLE
Bug found/ bug gefunden 

### ISSUE THREAD

**Issue by DarthZocker** [Mogli12/VehicleControlAddon#527]
First of all. German or english? 
Second. Thanks to mogli for the awesome mod. 
Now to the bug. 
I use to play a the saitek logitech side panel. And if you use the joystick of it for the frontloader of the tracor, i cant move the tool up and down. The frontloader itself its moveable and also the tools moving open and close. In other vehicles (front, wheel, telescopeloader) everythink works fine. 
In short 
Tractor frontloader up and down (keys j and n) works with side panel (y axis) 
Tractor frontloader tool up and down (keys k and m) not works with side panel (x axis)

Thanks to you

Greetings DZ

**Comment by Mogli12** [comment]
Hi,

I guess that this is a problem of your inputBinding.xml. I cannot see the connection to VCA,

Best regards

**Comment by DarthZocker** [comment]
I checked this. 
New game without the mod - no problem
New game with your mod - the problem
New game and standard input bindings without the mod - no problem
New game and standard input bindings with your mod - there is the bug again

**Comment by Mogli12** [comment]
Hi,

This a problem with your inputBinding.xml. Please read the [README.md](https://github.com/Mogli12/VehicleControlAddon/blob/master/README.md).

Thanks
